src/Data/Maybe.agda

```diff
@@ -56,6 +57,29 @@ monadPlus {f} = record
   nothing ∣ y = y
   just x  ∣ y = just x
 
+
+------------------------------------------------------------------------
+-- Other associated operations
+
+fromMaybe : {ℓ : Level} {A : Set ℓ} (default : A) (try : Maybe A) → A
+fromMaybe def (just x) = x
+fromMaybe def nothing  = def
+
+listToMaybe : {ℓ : Level} {A : Set ℓ} → List A → Maybe A
+listToMaybe []       = nothing
+listToMaybe (x ∷ xs) = just x
+
+maybeToList : {ℓ : Level} {A : Set ℓ} → Maybe A → List A
+maybeToList (just x) = [ x ]
+maybeToList nothing  = []
+
```

Comment:
Note that this is in `Data.List.Base` under the name [`fromMaybe`](https://agda.github.io/agda-stdlib/Data.List.Base.html#3476)